### PR TITLE
Correcting name of module

### PR DESCRIPTION
**Pull request by AmitThakkar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ A full example leveraging other community modules is contained in the [examples/
 
 ```hcl
 data "aws_eks_cluster" "cluster" {
-  name = module.eks.cluster_id
+  name = module.my-cluster.cluster_id
 }
 
 data "aws_eks_cluster_auth" "cluster" {
-  name = module.eks.cluster_id
+  name = module.my-cluster.cluster_id
```
Comment (max-rocket-internet) [inline README.md]: But it's called `my-cluster` on line 40?

```
module "my-cluster" {
```

Comment (barryib) [reply README.md]: Yep. That's why @AmitThakkar fixed the referenced module name in datasources. Or you have to name your module `module "eks" { ... }` on line 40 to make the exemple works.

**Comment by barryib** [comment]
@AmitThakkar Please update your branch.

**Comment by AmitThakkar** [comment]
@barryib Updated!

File: README.md (hunk 1 above)
Comment (max-rocket-internet) [reply README.md]: Ah yes, I must have been half asleep.

**Pull request merged**